dao/src/main/java/cz.muni.fi.pa165.soccermanager/entity/Player.java

```diff
@@ -1,14 +1,123 @@
 package cz.muni.fi.pa165.soccermanager.entity;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+import javax.persistence.*;
+import java.util.Date;
 
+/**
+ * @author Jan Novak
+ */
 @Entity
 public class Player {
 
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Temporal(TemporalType.DATE)
+    @Column(nullable = false)
+    private Date born;
+    private int position;
```

Comment:
I think position should be Enum (Goalkeeper, Defender, Midfielder, Attacker, or more specific) or at least String